lib/generators/module_component/templates/app/controllers/controller.rb.erb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module <%= class_name %>
+  module V0
+    class <%= class_name %>Controller < ApplicationController
+      skip_before_action :authenticate
+
+      def index
+        message = service.get_message
+        render json: <%= class_name %>Serializer.new(message).serialized_json
+      end
+
+      private
+
+      def service
+        Service.new
```

Comment:
I see it's only called once, but should this be memoized?

Reply:
Yeah I think that makes sense. I mean it's only called once right NOW, but could be called more than once as the controller grows